include/deal.II/lac/vector.h

```diff
@@ -152,26 +145,23 @@ class Vector : public Subscriptor
    *
    * We would like to make this constructor explicit, but standard containers
    * insist on using it implicitly.
+   *
+   * @note Since Subscriptor objects do not copy state in their copy
+   * constructor the default constructor suffices.
```

Comment:
Why is that important? I would always expect that the (default) base class copy constructor is called for a defaulted constructor and that this base class constructor is doing whatever is sensible for the base class.

Reply:
I wondered about this when I looked through the patch, but couldn't convince myself that it is wrong. When you write
```
  Vector (const Vector<Number> &src) = default;
```
then this would call the copy constructor of the `Subscriptor` base class -- which is not what we want. 

On the other hand, if you have
```
  template <typename OtherNumber>
  Vector (const Vector<OtherNumber> &src)
      // no base initializer list here
  {
     .. copy vector elements...
  }
```
then this calls the default constructor of the `Subscriptor` base class, which is what we want.

In neither case is this something that needs to be documented, I think.

Reply:
The copy constructor of `Subscriptor` does the 'right' thing, though:

```cpp
Subscriptor::Subscriptor(const Subscriptor &)
  : counter(0)
  , object_info(nullptr)
{}

```

I wrote that note after I went back and checked. 

I thought about this some more and it seems odd to explicitly document that 'the copy constructor is not wrong', so I think I will remove the whole thing anyway.